src/index.js

```diff
@@ -0,0 +1,239 @@
+const state = {
+  temp: parseInt(document.querySelector('#display-temp').innerHTML),
+  sky: '',
+  city: '',
+};
+
+skyValues = {
+  sunny: ['01d', '01n', '02d', '02n'],
+  cloudy: ['03d', '03n', '04d', '04n', '50d', '50n'],
+  raining: ['09d', '09n', '10d', '10n', '11d', '11n'],
+  snowing: ['13d', '13n'],
+};
+
+/* || TEMPERATURE */
+
+// Change color of temperature, based on current temperature
+const changeTempColor = () => {
+  const currentTemp = document.querySelector('#display-temp');
+  if (state.temp >= 80) {
+    currentTemp.style.color = 'red';
```

Comment:
Style: This would better follow [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns) if you gave this a class name (say `hot`) and styled it in CSS.